### PR TITLE
TTK-16753: Add update model from 2.3.x to 2.4.x

### PR DESCRIPTION
**Pull request by mreyk**

### CODE REVIEW

File: doc/updateModel_2.3_to_2.4.php [hunk 1]
```diff
@@ -24,20 +30,69 @@ protected function configure()
...
-        $output->writeln('Series.hide initialed to false');
+        $this->initParameters($input, $output);
+        $this->output->writeln('************* Update model from 2.3.x to 2.4.x ***************');
+        $this->initSeriesNewFields();
+        $this->initMultimediaObjectNewFields();
+        $this->output->writeln('End updating model.');
     }
 
-    protected function initSeriesHide()
+    protected function initParameters($input, $output)
```
Comment (Yurujai) [inline doc/updateModel_2.3_to_2.4.php]: Can you use 
**    protected function initialize(InputInterface $input, OutputInterface $output)

Comment (mreyk) [reply doc/updateModel_2.3_to_2.4.php]: @Yurujai done in commit 50edd1f

File: doc/updateModel_2.3_to_2.4.php [hunk 2]
```diff
@@ -30,14 +30,14 @@ protected function configure()
 
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        $this->initParameters($input, $output);
+        $this->initialize($input, $output);
```
Comment (Yurujai) [inline doc/updateModel_2.3_to_2.4.php]: It's not necessary, Initialize is a predefined function that is executed before execute action. You can see the use on: 
https://github.com/campusdomar/PuMuKIT2/blob/2.4.x/src/Pumukit/CoreBundle/Command/CreateMMOCommand.php#L43

File: doc/updateModel_2.3_to_2.4.php (hunk 1 above)
Comment (Yurujai) [reply doc/updateModel_2.3_to_2.4.php]: @mreyk can you use php cs fixer please?

Comment (mreyk) [reply doc/updateModel_2.3_to_2.4.php]: @Yurujai , done in commit 25961c22c8e47e8b1e389557bd1f1bddb34d9109

**Comment by Yurujai** [comment]
Thank you!

**Pull request merged**